sherpa/astro/ui/tests/test_astro_session_image.py

```diff
@@ -0,0 +1,196 @@
+#
+#  Copyright (C) 2021
```

Comment:
```suggestion
#  Copyright (C) 2022
```
at least I think that's the current year. Or is it still March 2020, about day 968?